### DISCUSSION TITLE
page resizing

### DISCUSSION THREAD

**Discussion by dvd1221** [flet-dev/flet#1329] [Q&A]
### Question

I want objects to be resized proportionally to the size of the web page. How can I do this? I sized objects by page.height and page.width. For example:
    search_field = ft.TextField(height=page.height * 0.04, width=page.width * 0.45, on_change=search)
![image](https://user-images.githubusercontent.com/126869058/234014342-c5aedaf0-8b1c-40b4-8c20-96391fe97c96.png)
Full Screen

![image](https://user-images.githubusercontent.com/126869058/234014630-69394bd5-3526-475a-bfcd-86b665593d85.png)
Partial Screen

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
I guess you should update control sizes in [`page.on_resize`](https://flet.dev/docs/controls/page#on_resize) event handler. Also, look into using `Row` and `Column` controls with `expand` property (https://flet.dev/docs/controls/row#expanding-children).
